website/docs/sql/scalar_func/arrays.md

```diff
@@ -0,0 +1,45 @@
+# Array Functions
+
+This section describes functions and operators to inspect and transform `array` values. In the following, `array(T)` denotes an array type with element type `T`. 
```

Comment:
```suggestion
This section describes functions and operators to inspect and transform [`array` values](../datatype/array.md). In the following, `array(T)` denotes an array type with element type `T`. 
```
(Double check whether using backticks in links works)

Reply:
[2786cb2](https://github.com/tableau/hyper-db/pull/110/commits/2786cb2785ecf8a9e5dc2f2643c681f83b916534)